.github/workflows/cypress_test.yml

```
@@ -0,0 +1,15 @@
+name: Cypress Tests
+on: [push]
+jobs:
+  cypress-run:
+    runs-on: ubuntu-22.04
```

```suggestion
    runs-on: ubuntu-22.04
        defaults:
      run:
        working-directory: cypress
```

this requires you to move the package.json to /cypress directory